build.gradle

```diff
@@ -3362,6 +3362,7 @@ project(':jmh-benchmarks') {
   tasks.withType(JavaCompile) {
     // Suppress warning caused by code generated by jmh: `warning: [cast] redundant cast to long`
     options.compilerArgs << "-Xlint:-cast"
+    options.compilerArgs << "-Xlint:unchecked"
```

Comment:
This should be included by `-Xlint:all`, right? 

https://github.com/apache/kafka/blob/trunk/build.gradle#L129

Reply:
Happy to remove it -- it seems something is off with my local setup -- I don't see the full warn output as you do, and adding this flag did make a difference for me... so 🤷